src/main/groovy/com/getkeepsafe/dexcount/sdkresolver/SdkResolver.groovy

```diff
@@ -25,12 +25,13 @@ import static com.android.SdkConstants.ANDROID_HOME_ENV
 import static com.android.SdkConstants.FN_LOCAL_PROPERTIES
 import static com.android.SdkConstants.PLATFORM_WINDOWS
 import static com.android.SdkConstants.SDK_DIR_PROPERTY
-import static com.android.SdkConstants.NDK_DIR_PROPERTY 
+import static com.android.SdkConstants.NDK_DIR_PROPERTY
 import static com.android.SdkConstants.currentPlatform
 
 /**
  * This was lifted and modified from Jake Wharton's SDK Manager gradle plugin:
  * https://github.com/JakeWharton/sdk-manager-plugin
+ * In order to keep original source intact, let's keep this in Groovy
```

Comment:
It would be an interesting exercise to port this to Kotlin, particularly as the source repo is no longer maintained.

Not that I'm saying we _should_ - IMO this is fine in Groovy.